recipes/lazypredict/LICENSE

```diff
@@ -0,0 +1,22 @@
+MIT License
```

Comment:
The license file is included in the source distribution so no need to manually include it here. Please remove this file.